src/test/rmc-dependency-test/diamond-dependency/run-dependency-test.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
```

Comment:
https://stackoverflow.com/questions/21612980/why-is-usr-bin-env-bash-superior-to-bin-bash

Reply:
Changed!